src/db.ts

```diff
@@ -67,10 +67,6 @@ class Database {
     this.db.serialize(() => {
       this.createTables();
       this.migrate();
-      // If database newly created, populate with default data
-      if (!exists) {
-        this.populate();
-      }
```

Comment:
Was this the only default data in the database?

Reply:
Yes.